modules/oxide/src/less/theme/components/toolbar/toolbar.less

```diff
@@ -155,3 +155,23 @@
   display: flex;
   align-items: center;
 }
+
+.tox-focusable-wrapper {
+  pointer-events: none;
+
+  .tox-textfield-size {
+    pointer-events: all;
+  }
+
+  &:focus {
+    background-color: @toolbar-button-bespoke-focus-background-color;
+    border-radius: @control-border-radius;
+    border-style: solid;
+    border-width: 1px;
+    &:extend(.tox .tox-textfield:focus);
+  }
+
+  &:has(input:focus)::after {
```

Comment:
I think you can use `focus-within` to avoid `has` though we use `has` inside tinymce so not sure it's a big problem there was some complains of old Chrome versions not working for a while.

Reply:
this has been removed